tests/integration/test_core.py

```diff
@@ -41,7 +48,7 @@ async def test_build_and_deploy(ops_test: pytest_operator.plugin.OpsTest, applic
     await ops_test.model.wait_for_idle()
     for unit in ops_test.model.applications[application_name].units:
         assert (
-            unit.workload_status == ops.model.BlockedStatus.name
+            unit.workload_status == ops.model.BlockedStatus.name  # type: ignore
```

Comment:
Is it worth including a comment with the reason for disabling?

Reply:
Fixed, thanks!